Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -358,8 +358,8 @@ void ProjectData::parseLinearSolvers(BaseLib::ConfigTree const& config)
         BaseLib::insertIfKeyUniqueElseError(
             _linear_solvers,
             name,
-            MathLib::createLinearSolver<GlobalMatrix, GlobalVector,
-                                        GlobalLinearSolver>(&conf),
+            std::unique_ptr<GlobalLinearSolver>(
+                new GlobalLinearSolver("", &conf)),
```

Comment:
I see. Thanks. Maybe some one more knowledgeable about PETSc solver creation could have a look on this; @chleh ?